### PR TITLE
Dispatch custom event from bookreader for ia-book-actions

### PR DESCRIPTION
**Pull request by nsharma123**
https://webarchive.jira.com/browse/WEBDEV-5711

Description: ia-book-actions uses new feature call `loan-auto-renew` where we need to send a specific pageChanged event when user clicked on new page.

### CODE REVIEW

File: src/BookReader.js [hunk 1]
```diff
@@ -1279,7 +1279,8 @@ BookReader.prototype.updateFirstIndex = function(
   if (this.options.initialSearchTerm && !suppressFragmentChange) {
     this.suppressFragmentChange = false;
   }
-  this.trigger('pageChanged');
+
+  this.trigger(BookReader.eventNames.pageChanged, { hasPageChanged: true });
```
Comment (iisa) [inline src/BookReader.js:1283]: Q: when page changes - `hasPagedChanged` is always true. feels like we do not need that custom value.

Comment (nsharma123) [reply src/BookReader.js:1283]: `BookReader.eventNames.pageChanged` actually being called from multiple places and even execute when you reload the page as well. so I need to catch specific event when user actually Click on any book page or change the page using br controls. so I found this function is appropriate from the actual fragementChange event does when user interact.

Comment (iisa) [reply src/BookReader.js:1283]: resolving to re-think abstraction

**Review by iisa — APPROVED** [review]
looks good. linting needs update - missing `;`

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@internetarchive/bookreader",
-  "version": "5.0.0-53",
+  "version": "5.0.0-53-alpha2",
```
Comment (iisa) [inline package.json:3]: blocker: please remove before merge

**Pull request merged**